projects/collection-editor-library/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@project-sunbird/sunbird-collection-editor-v9",
-  "version": "4.7.12",
+  "version": "4.7.13",
```

Comment:
@dhruvarya9654 please updated the version to 4.7.14.

Reply:
Updated to 4.7.14